CONTRIBUTING.md

```diff
@@ -123,9 +123,7 @@ target 'sample' do
 
   # ... react native config
 
-+  pod 'Sentry/HybridSDK', :path => '../../../sentry-cocoa'
-+  pod 'SentryPrivate', :path => '../../../sentry-cocoa/SentryPrivate.podspec'
-
+ pod 'Sentry/HybridSDK', :path => '../../../sentry-cocoa'
```

Comment:
```suggestion
 pod 'Sentry/HybridSDK', :path => '../../../../sentry-cocoa'
```

because the original path resolves to the root of sentry-react-native repo

Reply:
Good catch, thank you!